IPFS gateway change up (#187)
protocol/launchpad: docs/ipld/ipld-and-ipfs.md

#### How IPFS Deals With Files | IPFS Camp 2019 Workshop – Alan Shaw & Mikeal Rogers

> In this talk by Alan Shaw, He goes over a useful introduction to IPFS and file-based DAGs.

Reviewer: ```suggestion
In the following talk, Alan Shaw goes over a useful introduction to IPFS and file-based DAGs.
```
